builtin/game/features.lua

```diff
@@ -27,6 +27,8 @@ core.features = {
 	get_light_data_buffer = true,
 	mod_storage_on_disk = true,
 	compress_zstd = true,
+	sound_params_start_time = true,
+	physics_overrides_version = 2,
```

Comment:
Do not suddenly introduce new types here.
This doesn't fit into the concept, for example `core.has_feature` only works with bools.

Reply:
Oh right. I totally forgot about that function. Sorry for introducing a new mess - requested in https://github.com/minetest/minetest/pull/13978#discussion_r1390254162 . Versioning seemed to be a good solution but `core.has_feature` would indeed not work on it.